src/object-properties.js

```diff
@@ -63,6 +63,7 @@ const properties = {
   style: {
     label: DEFAULTS.LABEL,
     font: {
+      fontFamily: DEFAULTS.FONT_FAMILY,
```

Comment:
Do we really need to set this one? Would be great if we can without so the theme could override this

Reply:
How can I give reference to it then? font family is a dropdown with an option of different fonts